### ISSUE TITLE
Logging tests are frequently failing

### ISSUE THREAD

**Issue by Tomasz-Smelcerz-SAP** [kyma-project/kyma#17606]
**Description**

The logging tests are frequently failing on [`kyma-upgrade-gardener-kyma2-to-main-reconciler-main`](https://status.build.kyma-project.io/job-history/gs/kyma-prow-logs/logs/kyma-upgrade-gardener-kyma2-to-main-reconciler-main) and [`kyma-upgrade-k3d-kyma2-to-main`](https://status.build.kyma-project.io/job-history/gs/kyma-prow-logs/logs/kyma-upgrade-k3d-kyma2-to-main) prowjobs.

**Expected result**

The tests are passing

**Actual result**

```
   1) Upgrade test tests
       Logging Tests:
         Istio Access Logs tests
           Should query Loki and verify format of Istio Access Logs:
     TypeError: Cannot read property 'values' of undefined
      at Object.verifyIstioAccessLogFormat (logging/loki.js:58:48)
      at processTicksAndRejections (internal/process/task_queues.js:95:5)
      at async Context.<anonymous> (logging/index.js:45:7) 
```

**See also**

- https://status.build.kyma-project.io/view/gs/kyma-prow-logs/logs/kyma-upgrade-k3d-kyma2-to-main/1663968646646468608
- https://status.build.kyma-project.io/view/gs/kyma-prow-logs/logs/kyma-upgrade-k3d-kyma2-to-main/1663908248077471744
- https://status.build.kyma-project.io/view/gs/kyma-prow-logs/logs/kyma-upgrade-k3d-kyma2-to-main/1664331099871907840
- https://status.build.kyma-project.io/view/gs/kyma-prow-logs/logs/kyma-upgrade-gardener-kyma2-to-main-reconciler-main/1664300838761271296
- https://status.build.kyma-project.io/view/gs/kyma-prow-logs/logs/kyma-upgrade-gardener-kyma2-to-main-reconciler-main/1664270639537590272

**Comment by chrkl** [comment]
PR #17619 added retries for querying the access logs from Loki. Will monitor the mentioned jobs over the next days.

**Comment by chrkl** [comment]
The stability improved significantly after merging the mentioned PR. Since then, we saw only one failure in the listed jobs. As the access log test will be changed or removed in the near future, I will not continue to investigate. Please re-open or create a new issue if this becomes a problem again.

**Comment by chrkl** [comment]
The issue has already been fixed with https://github.com/kyma-project/kyma/pull/17791. The test is still failing since it is deploying Kyma 2.16.0, which does not include this fix. We backported the fix to 2.16.x with https://github.com/kyma-project/kyma/pull/17823 to make the test green again asap.
